### DISCUSSION TITLE
Version 11.9.0

### DISCUSSION THREAD

**Discussion by metastable-void** [menhera-org/TabArray#213] [Announcements]
This release is focused on improving overall performance.

## New features
- Web request performance is vastly improved. Yes, performance is a feature.
- Version information part is now properly translated.

## Improvements and fixes

- Now IO operations are fully cached, and web requests and tab sorting should be faster.
- Preparation work for import cookies feature. ([#212](https://github.com/menhera-org/TabArray/issues/212))
- Translation updates.
- Dependencies updates.
- Bug fixes.

Download and install from: https://addons.mozilla.org/firefox/addon/container-tab-groups/

<hr /><em>This discussion was created from the release <a href='https://github.com/menhera-org/TabArray/releases/tag/v11.9.0'>Version 11.9.0</a>.</em>

**Comment by JohnLGalt** [comment]
Weird issue when using 11.8.x and now 11.9.0. Still only using on the new, rebuilt profiles when testing for 11.7.x stuff, I've deleted the old profiles completely.

When I middle click on a link in a web page, which opens automatically in a new tab, instead of opening the new tab right after the current one, it sometimes opens 2 tabs away, sometimes 1 tab away.

However, when I actually click on that new tab, sometimes it remains where it was opened, and sometimes it automatically moves back adjacent (after) the tab it came from.

It doesn't break anything, and I've seen no real issues with interaction - it's just ... weird.

**Reply by metastable-void** [reply]
Does this build fix your problem?
[ctg-11.10.0.100-1684420201-d73e297.xpi.zip](https://github.com/menhera-org/TabArray/files/11513314/ctg-11.10.0.100-1684420201-d73e297.xpi.zip)

**Reply by JohnLGalt** [reply]
I'm just now seeing this build, I'll give it a shot and report back.
